pkgs/servers/samba/4.x.nix

```diff
@@ -57,6 +58,8 @@ stdenv.mkDerivation rec {
     ./patch-source3__libads__kerberos_keytab.c.patch
     ./4.x-no-persistent-install-dynconfig.patch
     ./4.x-fix-makeflags-parsing.patch
+    # Backport, should be removed for version 4.14
+    ./0001-lib-util-Standardize-use-of-st_-acm-time-ns.patch
```

Comment:
Can you use fetchpatch instead? This does not add the patch to nixpkgs and it can be more easily connected to the upstream one.

Reply:
No as mentioned in the description the patch needs to be adapted for the 4.13 branch.